js/logos.json

```diff
@@ -322,4 +322,8 @@
 	"title": "Globacom Limited",
 	"filename": "glo",
 	"category": "Telecommunication / ISP"
-}]
+}, {
+	"title": "Gigalayer",
+	"filename": "gigalayer",
+	"category": "Hosting Company"
```

Comment:
Hey @ladaposamuel, can we categorize Gigalayer as a `Software / ICT` company? `Hosting company` is not immediately clear to a non-tech savvy person.

Reply:
Thank you,
Changed it to `Software / ICT`